ITE/1/README.adoc

```diff
@@ -0,0 +1,925 @@
+
+. **<<final, Final Status>>** - When the implementation is complete
+  and all changes have been published or otherwise incorporated into the
+  appropriate code repositories, the ITE status is changed to "<<final, Final>>"
+  (or in some cases "<<active, Active>>"). The ITE is done.
+
+. **<<maintenance, Maintenance>>** - At some later date, the ITE may need to be
+  updated with minor changes and clarifications.  As <<Sponsor>> of the ITE,
+  Andrea makes changes as needed or hands off sponsorship to someone else.
+  Updates follow the same basic ITE workflow.  For extensive changes or
+  additions, Andrea will start a whole new ITE instead of updating the original
+  ITE.  This new ITE might expand on the orginal or might <<replaced, replace>>
+it.
+
+. **<<Other ITE Outcomes, Other Outcomes>>** - Not all ITEs will be accepted and finalized.
+  Other possible outcomes include "<<rejected, Rejected>>",
+  "<<deferred, Deferred>>", "<<withdrawn, Withdrawn>>".
+
+IMPORTANT: The above is only a high-level overview of the ITE workflow.
+The full and complete description of the ITE workflow is provided below.
+Read the full description below before starting a ITE.
+
+[[start]]
+==== Start with an idea for in-toto
+
+The ITE process begins with a new idea for in-toto.
+A single ITE should contain a single key proposal or new idea.
+The more focused the ITE, the more successful it tends to be.
+The ITE editors reserve the right to reject potential ITEs
+if they appear too unfocused or too broad.
+If in doubt, sponsors should split their ITE into several well-focused ones.
+
+[NOTE]
+====
+Enhancements or patches which have a smaller impact often don’t need an ITE.
+These can be handled via the regular in-toto development workflow with a GitHub 
+issue and/or pull request to the appropriate repository.
+
+An ITE may still have an accompanying GitHub issue as a placeholder.  This
+might be useful, for example, if implementation of the ITE is later found to
+have introduced a bug; being able to link the `regression`-labelled GitHub issue
+to the placeholder is valuable for tracking purposes. In such a case be sure
+to specify a "<<header-issue, issue>>" section.
+
+====
+
+==== Find a Sponsor
+
+Each ITE must have a "<<Sponsor>>" -- someone who writes the ITE using the
+style and format described below, shepherds the discussions in the appropriate
+forums, and attempts to build community consensus around the idea. The ITE
+Sponsor should first attempt to ascertain whether the idea is ITE-able.
+Posting to the in-toto-dev@googlegroups.com mailing list or opening an issue in
+the specification repository footnoteref:[repo] is
+the best way to go about this.
+
+[[discussion]]
+==== Discuss the idea with the community
+
+Vetting an idea publicly before going as far as writing a ITE is meant
+to save the potential sponsor time. Many ideas have been brought
+forward for changing Jenkins that have been rejected for various
```

Comment:
Jenkins?

Reply:
@SantiagoTorres: Before you change this in your fork, please take a look at mine and cherry pick instead (see https://github.com/lukpueh/ITE/commit/91d48a67c77a15cf5096163910c5dd648a63c7f6)